### PR TITLE
Rough in some techincal concepts

### PR DESCRIPTION
**Pull request by dstufft**
I was bored and decided to rough in a few of the technical concepts mentioned in #22. Feel free to reject or copy/paste text from here if it's at all useful to y'all.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -36,6 +36,9 @@ navigation:
 - text: Strict Transport Security
   url: hsts/
   internal: true
+- text: Techincal Concepts
```
Comment (syndbg) [inline _config.yml]: Just a small typo

**Comment by alex** [comment]
I'm not sure if anyone cares, but most of the other documents use "line per paragraph" as opposed to "line break at 80 columns"

**Comment by alex** [comment]
Overall this is some great material, thanks!

**Comment by dstufft** [comment]
Force of habit, I reflowed the paragraphs to break at the paragraph instead of at 80 columns.

**Comment by konklone** [comment]
> Force of habit, I reflowed the paragraphs to break at the paragraph instead of at 80 columns.

Thanks - yeah, for Markdown I've seen the 80-column limit do some weird stuff with links and lists and stuff. I'd prefer one line per paragraph.

This is a great start for this page, thank you @dstufft!

**Pull request merged**